File: api/DTOs/Admin/CallRecordDto.cs
namespace api.DTOs.Admin
{
    public class CallRecordDto
    {
        public int Id { get; set; }
        public string PersonType { get; set; }
        public string PersonId { get; set; }
        public string Name { get; set; }
        public string Email { get; set; }
        public string Subject { get; set; }
        public string MobileNo { get; set; }
        public string CategoryRef { get; set; }
        public DateTime CreatedOn { get; set; }
        public string Username { get; set; }    
        public DateTime ConcludedOn { get; set; }
        public string Status {get; set;}
    }
}

File: client/src/app/_dtos/admin/callRecordDto.ts
import { ICallRecordItem } from "src/app/_models/admin/callRecordItem";

export interface ICallRecordDto
{
     id: number;
     gender: string;
     userHistoryHeaderId?: number;
     checked?: boolean;
     source: string;
     categoryRef: string;
     categoryName: string;
     resumeId: string;
     name: string;
     personId?: number;
     personType: string;
     emailId: string;
     alternateEmailId: string;
     mobileNo:string;
     alternatePhoneNo: string;
     applicationNo?: number;
     createdOn: Date;
     concluded?: boolean;
     concludedByName: string;
     status: string;
     userName: string;
     errorMessage: string;
     callRecordItems: ICallRecordItem[];
}